### PR TITLE
Introduction to Digital Twin Technology

### LINKED ISSUE #694: Introduction to Digital Twin Technology
### Brief Summary: 
Digital twin technology is a concept that integrates software analytics, artificial intelligence and special network graphs to create digital simulation models that replicate physical assets (physical twin). It performs important functions such as optimization of operations in manufacturing processes and driving business value. This article gives readers meaningful information regarding this new technology. 

### Key Takeaways: 
1. What is a Digital Twin?
2. Why is Digital Twin Technology Important?
3. Characteristics of Digital Twin Technology
4. How Digital Twin Technology Works
5. Applications of Digital Twin Technology
6. The Future of Digital Twin Technology

### References: 
file:///C:/Users/ADMIN/AppData/Local/Temp/an-introduction-to-digital-twin-development.pdf

### PR DESCRIPTION
**Pull request by mbaabuones**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [ ] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

This PR fixes #694

### CODE REVIEW

**Comment by mbaabuones** [comment]
@hectorkambow Im very sorry for having 10 commits in this PR. This is contrary to your initial request. I must admit I am facing a challenge especially when creating new branches. Every time I create a new PR, I always find files files from my other PR(s). I am forced to delete the unrequired files. Im kindly requesting for your guidance on how I can submit all my files in a single commit without affecting other PRs.

**Comment by hectorkambow** [comment]
> @hectorkambow Im very sorry for having 10 commits in this PR. This is contrary to your initial request. I must admit I am facing a challenge especially when creating new branches. Every time I create a new PR, I always find files from my other PR(s). I am forced to delete the unrequired files. Im kindly requesting for your guidance on how I can submit all my files in a single commit without affecting other PRs.

👋  Not a problem - and thank you for asking for help. 

I don't recall if I've shared this video with you - but these short videos really helped me out a lot when I was brand new to GitHub and Git flows. 

- [GIT: Working with Branches](https://www.youtube.com/watch?v=JTE2Fn_sCZs&t=127s)
- [How to Upload Folder and Files in Github Repositories](https://www.youtube.com/watch?v=wXGa0bDvFWA&t=159s)
- [1.6: Cloning Repo and Push/Pull - Git and GitHub for Poets](https://www.youtube.com/watch?v=yXT1ElMEkW8&t=136s)
- [1.3: Forks and Pull Requests - Git and GitHub for Poets](https://www.youtube.com/watch?v=_NrSWLQsDL4)
- [Git & GitHub Tutorial for Beginners #11 - Collaborating on GitHub](https://www.youtube.com/watch?v=MnUd31TvBoU&t=513s)

These will explain how to work with branches (each new article - in your text editor should be on a new branch) that way when you create a PR it only uploads that one branch.

Hope this helps - please feel free to ping me if you have any other questions. 🚀

File: articles/introduction-to-digital-twin-technology/index.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+Digital twin technology is a concept that integrates the internet of things (IoT), software analytics, artificial intelligence, and special network graphs to create digital simulation models that replicate physical assets (physical twin).
+
+### What is a Digital Twin?
```
[the remark this answers is not in the document]
Comment (lalith1403) [reply articles/introduction-to-digital-twin-technology/index.md:3]: I think you need to emphasize the twin simulated technology part. That way, the entire article makes sense.

**Review by lalith1403 — CHANGES_REQUESTED** [review]
Amazing article. It needs a bit of restructuring. I have given a few suggestions. The fact that it is a digital twin of a live process is not clear in the article. You need to emphasize that and then proceed with the characteristics. Probably include an application in the beginning. That way, the reader's attention is retained. Also, a Grammarly check after the edits would ensure minimal errors. 

Also, check for lengthy sentences on hemingwayapp.com. Good job! Cheers.

**Comment by hectorkambow** [comment]
plagiarism check done

File: articles/introduction-to-digital-twin-technology/index.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
+Digital twin technology is a concept that integrates the internet of things (IoT), software analytics, artificial intelligence, and special network graphs to create digital simulation models that replicate physical assets (physical twin).
+
+### What is a Digital Twin?
+A digital twin is a digital representation of actual and potential assets, people, processes, systems, devices, and places. This representation provides the dynamics and elements of how an IoT device lives and operates. 
+
+Continuous learning and updates enable the digital twin to provide the real-time position, condition, or status of the real objects. This pairing of the real world and the digital representation enable organizations to monitor systems, develop plans and anticipate problems before they occur.
```
[the remark this answers is not in the document]
Comment (mbaabuones) [reply articles/introduction-to-digital-twin-technology/index.md]: I have rephrased it.

File: articles/introduction-to-digital-twin-technology/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mbaabuones) [reply articles/introduction-to-digital-twin-technology/index.md:7]: I have revised the introduction section.

File: articles/introduction-to-digital-twin-technology/index.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+![Digital Twin](/engineering-education/introduction-to-digital-twin-technology/digital-twin.jpg)
+
+[Image Source: IDGESG](https://images.idgesg.net/images/article/2018/06/digital-twins_woman-in-profile_ai_mirror_duplicate_duo_pair-100760562-large.jpg)
+
+### Why Digital Technology is Important
+Digital technology should be considered by organizations for various reasons. It enables companies to develop a digital footprint of the entire product life cycle. This ensures that they have an in-depth understanding of the actual product and the process that was used to design the product. The digital twin created by a company enables it to drive value through improved models and reduced defects. 
+
+A Digital twin enables a company to detect problems in a product or manufacturing process. Through digital technology, outcomes can be predicted effectively with a high level of accuracy. With this technology, there is faster  realization of value than before.
```
[the remark this answers is not in the document]
Comment (mbaabuones) [reply articles/introduction-to-digital-twin-technology/index.md]: Done.

File: articles/introduction-to-digital-twin-technology/index.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+Continuous learning and updates enable the digital twin to provide the real-time position, condition, or status of the real objects. This pairing of the real world and the digital representation enable organizations to monitor systems, develop plans and anticipate problems before they occur. 
+
+![Digital Twin](/engineering-education/introduction-to-digital-twin-technology/digital-twin.jpg)
+
+[Image Source: IDGESG](https://images.idgesg.net/images/article/2018/06/digital-twins_woman-in-profile_ai_mirror_duplicate_duo_pair-100760562-large.jpg)
+
+### Why Digital Technology is Important
+Digital technology should be considered by organizations for various reasons. It enables companies to develop a digital footprint of the entire product life cycle. This ensures that they have an in-depth understanding of the actual product and the process that was used to design the product. The digital twin created by a company enables it to drive value through improved models and reduced defects.
```
[the remark this answers is not in the document]
Comment (mbaabuones) [reply articles/introduction-to-digital-twin-technology/index.md]: I have modified this part.

File: articles/introduction-to-digital-twin-technology/index.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+Digital technology enhances the customization of digital production modules. This enables manufacturers to modify their models. Modularity in digital technology enables manufacturers to identify areas that need improvement.
+
+### How Digital Technology Works
+The Digital twin a of a manufacturing process can give us a good overview of how digital technology works. In this case, the digital twin provides a virtual representation of what happens in the factory in real-time. The following diagram shows the digital twin model of an actual manufacturing process.
+
+![How Digital Twin Technology Works](/engineering-education/introduction-to-digital-twin-technology/how-digital-twin-technology-works.jpg)
+
+[Image Source: Deloitte University Press](https://www2.deloitte.com/content/dam/insights/us/articles/3833_Industry4-0_digital-twin-technology/figures/3833_Figure1.png)
+
+The digital twin model is achieved through six main features-sensors, actuators, data, integration, analytics, and the digital twin.
```
[the remark this answers is not in the document]
Comment (mbaabuones) [reply articles/introduction-to-digital-twin-technology/index.md]: I have restructured the article. I have also revised the section on 'how digital twin technology works'.

File: articles/introduction-to-digital-twin-technology/index.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+### How Digital Technology Works
+The Digital twin a of a manufacturing process can give us a good overview of how digital technology works. In this case, the digital twin provides a virtual representation of what happens in the factory in real-time. The following diagram shows the digital twin model of an actual manufacturing process.
+
+![How Digital Twin Technology Works](/engineering-education/introduction-to-digital-twin-technology/how-digital-twin-technology-works.jpg)
+
+[Image Source: Deloitte University Press](https://www2.deloitte.com/content/dam/insights/us/articles/3833_Industry4-0_digital-twin-technology/figures/3833_Figure1.png)
+
+The digital twin model is achieved through six main features-sensors, actuators, data, integration, analytics, and the digital twin. 
+
+- **Sensors:** Sensors are positioned at various points of the physical manufacturing process to collect data. Some of the important dimensions of data collected include works in progress (including speed, thickness, and color quality), environmental conditions of the factory, and behavioral characteristics of machines.
```
[the remark this answers is not in the document]
Comment (mbaabuones) [reply articles/introduction-to-digital-twin-technology/index.md]: Done.

**Comment by mbaabuones** [comment]
@lalith1403 Thank you for your comments and suggestions. I have finished working on the changes.

**Comment by lalith1403** [comment]
@hectorkambow There are merge conflicts. Can you suggest the next set of actions? The changes made are excellent. This is ready for the final round of review.

**Comment by hectorkambow** [comment]
There is a merge conflict with this PR 
If you could please migrate the most updated .index file and all image to a new PR to be merged. (that should be a separate branch)
Thank you

**Comment by mbaabuones** [comment]
> There is a merge conflict with this PR
> If you could please migrate the most updated .index file and all image to a new PR to be merged. (that should be a separate branch)
> Thank you
> image
> 
> The image was taken from our [FAQ](https://github.com/section-io/engineering-education/blob/master/new_contributors/FAQ.md)

@hectorkambow Done.

**Pull request closed without merging**